runtime/development/src/migrations.rs

```diff
@@ -10,7 +10,42 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
 
+use sp_core::parameter_types;
+parameter_types! {
+	pub const CollatorReward: cfg_primitives::Balance = cfg_primitives::constants::CFG;
+	pub const AnnualTreasuryInflationPercent: u32 = 3;
+}
+
 /// The migration set for Development & Demo.
 /// It includes all the migrations that have to be applied on that chain.
-pub type UpgradeDevelopment1046 =
-	pallet_collator_selection::migration::v1::MigrateToV1<crate::Runtime>;
+pub type UpgradeDevelopment1046 = (
+	pallet_collator_selection::migration::v1::MigrateToV1<crate::Runtime>,
+	// v0 -> v1
+	pallet_multisig::migrations::v1::MigrateToV1<crate::Runtime>,
+	// v0 -> v1
+	pallet_balances::migration::MigrateToTrackInactive<super::Runtime, super::CheckingAccount, ()>,
+	// v0 -> v1
+	runtime_common::migrations::increase_storage_version::Migration<crate::Preimage, 0, 1>,
+	// v0 -> v1
+	pallet_democracy::migrations::v1::v1::Migration<crate::Runtime>,
+	// v0 -> v1
+	pallet_xcm::migration::v1::VersionUncheckedMigrateToV1<crate::Runtime>,
+	runtime_common::migrations::increase_storage_version::Migration<crate::PoolSystem, 0, 2>,
```

Comment:
FYI, we have now a legacy version for this: https://github.com/paritytech/substrate/pull/14311